### ISSUE TITLE
Create secret of type kubernetes.io/tls

### ISSUE THREAD

**Issue by xbglowx** [machinezone/configmapsecrets#8]
Is it currently possible to change the type to be `kubernetes.io/tls`?

**Comment by abursavich** [comment]
No, that's not currently supported. That sounds like a job for [SealedSecrets](https://github.com/bitnami-labs/sealed-secrets) to me, but I'll try to keep an open mind if you explain your use case.

**Comment by xbglowx** [comment]
I would like to use cert-manager/let's encrypt, along with this, to dynamically create mTLS certs with a predefined CA cert.
